go-controller/pkg/cni/ovs.go

```diff
@@ -135,12 +151,16 @@ func doPodFlowsExist(mac string, ifAddrs []*net.IPNet) bool {
 		tables []int
 	}
 
-	// Query the flows by mac address for in_port_security
+	// Query the flows by mac address for in_port_security and OF port
```

Comment:
nit: update the comment

`//Query the flows by mac address and ofport for in_port_security`